subjects/DPxAI/i-win-arguments/README.md

```diff
@@ -0,0 +1,158 @@
+
+```js
+let myFirstFunction = (arg1, arg2) => {
+  //<-arg1 and arg2 are inputed in between the parens
+  console.log(arg1, arg2);
+  //            ↖   arg1 and arg2 are "transfered" to be the args of console.log()
+};
+// Now we call the function
+myFirstFunction("first arg", "second arg");
+// "first arg"
+// "second arg"
+```
+
+For more args, you will need to simply repeat the same process! Comma `,` then the other argument and it goes on.
+
+> Please note that you can name your arguments however you please. Just make sure that you reuse the proper name inside the scope of your function.
+
+### Return value
+
+In addition to accepting arguments, functions can also `return` values.
+
+Return values are the `outputs` that a function provides after completing its task.
+
+We are now going to adapt `myFirstFunction` so that it `returns` a value instead of just `logging it`.
+
+```js
+let myFirstFunction = (arg1) => {
+  return arg1; // the function now returns the value of arg1
+};
+```
+
+Now if the function is called, it returns the value of `arg1`:
+
+```js
+let result = myFirstFunction("using my first return");
+console.log(result); // "using my first return"
+```
+
+But let's say we want to change what the function `returns`. Now, instead of modifying `myFirstFunction`, we just need to modify `the argument` in `the function call`.
+
+```js
+let anotherResult = myFirstFunction("another return");
+console.log(anotherResult); // "another return"
+```
+
+#### **`Prompt Example`**:
+
+- "Can you guide me through creating and using a JavaScript function that takes multiple arguments, starting from a basic function without arguments, then adding single and multiple arguments ?"
+
+### Instructions
+
+#### Task 1:
+
+You are the general's aide responsible for transmitting communications to the other RoboGuards.
+
+1- Create the `battleCry` Function:
+
+- This function should take `one argument` and display it in the `console`.
+- The battlefield is vast, so ensure that `the argument is uppercased` before displaying it.
+
+2- Create the `secretOrders` Function:
+
+- Sometimes, communications need to be given quietly.
+
+- This function will do the `same` as `battleCry`, except `it will lowercase` the argument before sending it.
+
+> hint: you remmeber methods?
+
+#### Task 2:
+
+As the leader of the RoboGuard forces, you're not just preparing for battle—you're also forming dynamic duos of robots to work together on special missions.
+
+1- Create the `duos` Function:
+
+- This function will take `two arguments`, representing the **names** of **two robots**.
+- It will `log them` together with an **and** and an **exclamation mark**.
+
+2- Create the `duosWork` Function:
+
+- This function will take `three arguments`: the **names** of two robots and the **task** they will perform together.
+
+- It will `log them` together in a sentence describing their task.
```

Comment:
It's not clear what should be logged from this info only: am I missing some information in the subject? :sweat_smile:

Reply:
Addressed.